### ISSUE TITLE
Should it save content if Save Draft or Publish aren't hit?

### ISSUE THREAD

**Issue by MichaelArestad** [MichaelArestad/Press-This#83]
> If I don't click "save draft" or "publish" does it save as a draft or will I trash it if I exit out? I can't tell

via @krogsgard

I don't believe it does and I'm not sure it should. Thoughts?

**Comment by danielbachhuber** [comment]
Auto-save so users never lose content.

**Comment by stephdau** [comment]
@azaozz recently added a standard "leaving this page" notif, as done on FB, etc. I believe it kicks in once the content starts changing.

![screen shot 2015-02-18 at 20 35 00](https://cloud.githubusercontent.com/assets/1891/6260501/cc5bca76-b7ad-11e4-998f-073465b37daf.png)

**Comment by stephdau** [comment]
I think auto-save, if we go that route, would probably only be desirable if the user starts updating the content. Many users might start "pressing" something, and decide not to go ahead. I'd do whatever core does with its own editor, really.

**Comment by MichaelArestad** [comment]
Personally, I would prefer not to have a draft box full of unpublished posts, but like I said, I don't mind too much either way.

Well... even more full.

**Comment by krogsgard** [comment]
I wonder if a trash button would help someone figure this out as well? Perhaps duplicate, but would offer clarity.

**Comment by stephdau** [comment]
> I wonder if a trash button would help someone figure this out as well? Perhaps duplicate, but would offer clarity.

We used to have an x icon (close), but it was deemed redundant when coupled with OS based ways of closing a window. IIRC, we opted to keep the ui cleaner, instead of offering an option they already had. Not sure if a trash button would help much. This is probably a thing that we’ll find out with wider adoption and usage, and we’ll adapt to the demand. :)

**Comment by krogsgard** [comment]
Fwiw, Gmail uses the X and the trash can. To me the x doesn't signify deletion, but dismissal. https://cldup.com/sQcEQl_naI.png

**Comment by stephdau** [comment]
Indeed. Dismissing is what it is in our case, since the content is not saved, and we're currently not in favor of doing so. As everything else, users will tell us which one they want once it's released. Confirming leaving is currently what's done most often in similar sharing contexts, like FB, so we should be good for a base.
